cms/templates/js/course-outline.underscore

```diff
@@ -44,7 +44,8 @@ if (xblockInfo.get('graded')) {
     <div class="<%= xblockType %>-header">
         <% if (includesChildren) { %>
             <h3 class="<%= xblockType %>-header-details expand-collapse <%= isCollapsed ? 'expand' : 'collapse' %> ui-toggle-expansion"
-                title="<%= interpolate(gettext('Collapse/Expand this %(xblock_type)s'), { xblock_type: xblockTypeDisplayName }, true) %>">
+                title="<%= interpolate(
+				gettext('Collapse/Expand this %(xblock_type)s'), { xblock_type: xblockTypeDisplayName }, true) %>">
```

Comment:
Could you please adjust the style here a little bit to put the closing markers on new lines, such as:
```javascript
    title="<%= interpolate(
  		gettext('Collapse/Expand this %(xblock_type)s'), { xblock_type: xblockTypeDisplayName }, true
    ) %>"  // closes the "<%= from L47
>  // closes the <h3 from L46
```

You don't have to explicitly call out what lines close what, but doing this style consistently will make this a lot easier to read.